coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20171122-add-secondary-jobs-tables.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="Alexa Rust" id="20171122-add-secondary-jobs-tables">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <not>
+                    <tableExists tableName="secondary_add_jobs"/>
+                </not>
+                <not>
+                    <tableExists tableName="secondary_delete_jobs"/>
+                </not>
+            </and>
+        </preConditions>
+        <createTable tableName="secondary_add_jobs">
+            <column name="dataset_system_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="store_id" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="job_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="step" type="SMALLINT">
```

Comment:
This is an innocent question/learning opportunity for me: what would the drawbacks be for using an ENUM here instead so that the meaning of 'step' is also stored in the database as opposed to just a number?

Reply:
step is insteaded to be a step in a collocation operation (or some other hypothetical operation). There can be an arbitrary number of steps in a job so an enum would not work.